src/Wallabag/UserBundle/Resources/translations/wallabag_user.de.yml

```diff
@@ -0,0 +1,11 @@
+# Two factor mail
+auth_code:
+    on: 'an'
+    mailer:
+        subject: 'Wallabag Authentifizierungcode'
```

Comment:
So is it correct that my PR wants to merge to master? Or should i point it to the 2.2 branch?